packages/shared/src/components/post/collection/CollectionPillSources.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import Pill from '../../Pill';
+import { SourceAvatar, SourceAvatarProps } from '../../profile/source';
+import { ProfilePictureGroup } from '../../ProfilePictureGroup';
+
+interface CollectionPillSourcesProps {
+  className?: string;
+  sources: SourceAvatarProps['source'][];
+  totalSources: number;
+}
+export const CollectionPillSources = ({
+  className,
+  sources,
+  totalSources,
+}: CollectionPillSourcesProps): ReactElement => {
+  const hasSources = !!sources?.length;
+
+  return (
+    <div className={classNames(className, 'flex relative flex-row gap-2')}>
+      <div className="relative">
```

Comment:
Do we still need this extra wrapper? I think we can go away without it. Also that the gap is not taking effect due to the elements being considered as only a single item.

Reply:
Actually the gap was also not needed, cleaned up.